qemu/tests/nic_teaming.py

```diff
@@ -80,6 +80,11 @@ def team_if_exist():
         team_exists_cmd = params.get("team_if_exists_cmd")
         return session_serial.cmd_status(team_exists_cmd, safe=True) == 0
 
+    host_br = params.get("netdst", "switch")
+    br_backend = utils_net.find_bridge_manager(host_br)
+    if not isinstance(br_backend, utils_net.Bridge):
```

Comment:
Can be simplified to
```
if params["netdst"] not in utils_net.Bridge().list_br():
```

Reply:
Done and test pass on my side, please help review it again. Thanks.